modules/cats-effect/src/main/scala/pureconfig/module/catseffect/package.scala

```diff
@@ -169,42 +137,34 @@ package object catseffect {
     *
     * @param conf The configuration to save
     * @param outputPath Where to write the configuration
-    * @param blocker the blocking context which will be used to load the configuration.
     * @param overrideOutputPath Override the path if it already exists
     * @param options the config rendering options
     * @return The return action will save out the supplied configuration upon invocation
     */
   def blockingSaveConfigAsPropertyFileF[F[_], A](
```

Comment:
We should remove the deprecated `saveConfigAsPropertyFileF` method now, since it has the same signature as this and its implementation doesn't correctly scope blocking operations.

Reply:
Removed deprecated methods and reused their names for blocking implementations.